integration_tests/models/schema_tests/timeseries_hourly.sql

```diff
@@ -1,5 +1,9 @@
-{{ dbt_date.date_spine('hour',
-        start_date=dbt_date.n_days_ago(10),
-        end_date=dbt_date.tomorrow()
+{% set now = modules.datetime.datetime.now().replace(minute=0, hour=0, second=0, microsecond=0) %}
```

Comment:
Maybe it needs to be:

`{% set now = modules.datetime.date.today() %}`

?

Reply:
(Looking at the BQ error in CI)

Reply:
Maybe, thanks! This is also related to https://github.com/calogica/dbt-date/pull/106 which I just merged. Once I figure out the rest of the BQ CI issues, I'll merge and release this early this week.